pkg/storage/fs/cephfs/connections.go

```diff
@@ -63,7 +63,7 @@ func newCache() (c *connections, err error) {
 		MaxCost:     usrLimit,
 		BufferItems: 64,
 		OnEvict: func(item *ristretto.Item) {
-			v := item.Value.(cacheVal)
+			v := item.Value.(*cacheVal)
```

Comment:
I would still check that the type is correct with the notation `v, ok := item.Value.(*cacheVal)` otherwise in production it may panic.